Workaround for problem that ant's java.specification.version reports 1.5
but that's not the default source for <javac> as of F7.  This will probably
be fixed by a cleaner upstream change, but for now, just force it.


diff -Naur postgresql-jdbc-8.2-505.src.orig/build.xml postgresql-jdbc-8.2-505.src/build.xml
--- postgresql-jdbc-8.2-505.src.orig/build.xml	2006-11-28 23:00:15.000000000 -0500
+++ postgresql-jdbc-8.2-505.src/build.xml	2007-04-24 13:31:29.000000000 -0400
@@ -112,7 +112,7 @@
     <available classname="org.postgresql.Driver" property="old.driver.present" />
     <fail message="Old driver was detected on classpath or in jre/lib/ext, please remove and try again." if="old.driver.present" />
 
-    <javac classpath="${srcdir}" srcdir="${srcdir}" destdir="${builddir}" debug="${debug}">
+    <javac classpath="${srcdir}" srcdir="${srcdir}" destdir="${builddir}" debug="${debug}" source="1.5">
       <!-- This is the core of the driver.  It is common for all versions. -->
        <include name="${package}/*.java" />
        <include name="${package}/core/**" />
@@ -386,7 +386,7 @@
   <target name="testjar" depends="jar">
     <fail message="JUnit could not be found in your classpath.  You must download and install it from http://junit.org to build and run the test suite." unless="junit" />
     <mkdir dir="${builddir}/tests"/>
-    <javac srcdir="${srcdir}" destdir="${builddir}/tests" debug="${debug}">
+    <javac srcdir="${srcdir}" destdir="${builddir}/tests" debug="${debug}" source="1.5">
       <include name="${package}/test/**" />
 
       <exclude name="${package}/test/jdbc2/optional/**" unless="jdbc2optionaltests" />
